aptos-move/e2e-benchmark/src/main.rs

```diff
@@ -138,6 +142,8 @@ fn main() {
         (257, EntryPoints::TokenV1MintAndTransferFT),
         (412, EntryPoints::TokenV1MintAndTransferNFTSequential),
         (368, EntryPoints::TokenV2AmbassadorMint { numbered: true }),
+        (520, EntryPoints::LiquidityPoolSwap { is_stable: true }),
```

Comment:
What does stable and unstable mean here? Which mode is generally more used/recommended in practice?

Reply:
oh it depends whether you are doing a swap between assets that should change in price (i.e. ETH and SOL), or whether you are swapping between stablecoins, which should have the same price/fixed exchange ratio